package.json

```diff
@@ -61,6 +61,7 @@
     "@types/chai": "^4.0.10",
     "@types/chai-as-promised": "^7.1.0",
     "@types/mocha": "^2.2.45",
+    "@types/nock": "^9.3.0",
```

Comment:
i believe you also need to add `nock`, not only `@types/nock`

Reply:
Hmm I didn't have any issues in my tests nor lint, nor in Travis, do you still think is worth it to include it ?

Reply:
yes, most likely `nock` was included as a dependency from another package, however the `@types/nock` package does not include it as a dependency. Add `nock` in here to have a specific version and not depend on side effects from other packages